documentation/plain_rules.md

```diff
@@ -45,7 +45,8 @@ from pattern
 * The *from* pattern must name the event as **ev** and the event stream from which take events must be **iotEvent**
 * A *type=* condition must be concatenated for avoiding mixing different kinds of entities
 
-The used entity's atrributes must be cast to `float` in case of being numeric (like  in the example). Alphanumeric values must be cast to `String`. Nested cast to string and to float is something we are analyzing, and could be unnecessary in a future version. Use it by now. All the attributes in the notification from Orion are available in the event object, **ev**,  like *ev.BlodPressure?* and *ev.id?*. A question mark is *necessary* for EPL referring ‘dynamic’ values.
+The used entity's attributes must be cast to `float` in case of being numeric (like  in the example). Alphanumeric values must be cast to `String`. Nested cast to string and to float is something we are analyzing, and could be unnecessary in a future version. Use it by now. All the attributes in the notification from Orion are available in the event object, **ev**,  like *ev.BlodPressure?* and *ev.id?*. A question mark is *necessary* for EPL referring ‘dynamic’ values.
```

Comment:
Moreover, after this change I understand that "All the attributes in the notification from Orion are available in the event object" may also mention that metadata are available in the event object.